packages/wrangler/src/workflows/commands/delete.ts

```diff
@@ -21,13 +19,8 @@ defineCommand({
 	},
 	positionalArgs: ["name"],
 
-	async handler(args, { config }) {
-		const accountId = await requireAuth(config);
-
-		await fetchResult(`/accounts/${accountId}/workflows/${args.name}`, {
-			method: "DELETE",
-		});
-
-		logger.info(`Workflow "${args.name}" was successfully removed`);
+	async handler(args) {
+		logger.info("🚫 delete command not yet implement");
+		logger.log(`🚫 Workflow "${args.name}" NOT removed`);
```

Comment:
Let's not do this - our API should return a good error to the user that deletion is yet to be supported, so that when we implement it, this doesn't need to be reversed.

Reply:
😱 - we should just comment out this command for now, so no one can try and use it.

Reply:
The feedback I got from my team was "The command is already public, let's not remove it. Just show an error message." 😅

Reply:
Which one do you prefer?
1. Implement the API, throw an error on the API and gracefully manage it in Wrangler?
2. Delete the command from Wrangler all together until we have the API and functionality properly implemented?

Reply:
Even if we don't remove it from wrangler, I think we should hide it so it doesn't show up on the help message etc.

Reply:
> Even if we don't remove it, I think we should hide it so it doesn't show up on the help message etc.

I can live with this, I think it's fine to leave the command because removing would semantically suggest a breaking change in the cli

Reply:
I go for 2

Reply:
Hiding is fine, but can the other change be reverted & we add a better error to the API in the meantime?

Reply:
Once we update the API, we will add an improved error message to the it and update the Wrangler command accordingly.